appserver/admingui/payara-console-extras/src/main/resources/fish/payara/admingui/extras/Strings.properties

```diff
@@ -1,7 +1,7 @@
 #
 # DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
 #
```

Comment:
Can you also create a Spanish and Portuguese translation file asking Daniel and Veronica for the translation?